### PR TITLE
Prompt to upgrade google-play-services when < version we depend on

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

**Comment by wilsonpage** [comment]
Followup https://github.com/mozilla-magnet/magnet-client/issues/323

**Comment by wilsonpage** [comment]
@arcturus r?

File: android/app/build.gradle [hunk 1]
```diff
@@ -146,12 +146,22 @@ android {
...
-    compile project(':react-native-google-analytics-bridge')
     compile project(':react-native-linear-gradient')
     compile project(':magnet-scanner-android')
-    compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:23.0.1"
-    compile "com.facebook.react:react-native:+"  // From node_modules
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.android.support:appcompat-v7:23.0.1'
+    compile 'com.facebook.react:react-native:+'  // From node_modules
+    compile 'com.bugfender.sdk:android:0.+'
```
Comment (arcturus) [inline android/app/build.gradle:154]: Is this intended?

**Comment by arcturus** [comment]
LGTM, I think we need to remove the import in gradle for bugfender, when so, shiiiip it.

File: android/app/build.gradle (hunk 1 above)
Comment (wilsonpage) [reply android/app/build.gradle:154]: Ouch, nice catch! That was from a merge conflict.

**Comment by wilsonpage** [comment]
Closing, superseded by #323

**Pull request closed without merging**